addon/display/panel.js

```diff
@@ -100,7 +103,6 @@
         } else {
           info.wrapper.style.height = newHeight;
           newHeight = info.wrapper.offsetHeight;
-          info.wrapper.style.height = "";
```

Comment:
Why did you remove this line?

Reply:
This line removes the style on the wrapper element and I don't quite get why:

You can see it in the fiddle I linked in the bug report https://jsfiddle.net/xt64L72a/ (Set a breakpoint at this line and hit "run" )

Initially the wrapper has 90%, and it should always be 90%,
however when this line is executed, the 90% are removed, thus the wrapper sets it height to the height of its contents (panel and editor) and will stay at that height forever.

Reply:
The code is trying to convert a percentage height to a pixel height by first setting it, measuring the DOM, and then removing the style again. So clearing that is intentional.

Reply:
I understand the intention, but it seems to be executed on the wrong DOM element.
If the container already has a percentage value intentionally, it will be removed and is lost.

From my understanding the line where it is set first is not needed either since it should be set already, but I ran into issues during testing so I left it...

Reply:
Ok I do understand that setting it is required when calling setSize with arguments,
However it should not be removed.

Imagine you call setSize(500, "50%"):
The CodeMirror Container will be set to 50%,
this value will be converted to pixels and shoved into _setSize to resize the actual CoreMirror.

Then the container height (50%) will be removed. The Container won't change it's size since its children make up exactly those 50% it is supposed to be, but it will stay this big forever.
The Height of the container has been changed from dynamic 50% to whatever many pixels were 50% at the time the function was called...